packages/next/src/client/components/react-dev-overlay/ui/styles/base.tsx

```diff
@@ -15,18 +14,13 @@ export function Base() {
             'position: absolute;'
           }
 
-          --size-gap-half: 4px;
-          --size-gap: 8px;
-          --size-gap-double: 16px;
-          --size-gap-triple: 24px;
-          --size-gap-quad: 32px;
-
-          --size-font-11: 11px;
-          --size-font-smaller: 12px;
-          --size-font-small: 14px;
-          --size-font: 16px;
-          --size-font-big: 20px;
-          --size-font-bigger: 24px;
+          --size-font-11: var(--size-11);
```

Comment:
we should define the actual value if possible to avoid using the variable for variable.